src/lib/fuzzySearch.js

```diff
@@ -0,0 +1,31 @@
+import Fuse from "fuse.js";
+import { getIndexedContents } from "./github.js";
+
+/**
+ * Searches a indexed contents via the Fuse library
+ */
+export async function searchIndexedContents(query) {
+  const list = getIndexedContents();
+  var options = {
+    shouldSort: true,
+    findAllMatches: true,
+    includeMatches: true,
+    threshold: 0.6,
+    location: 0,
+    distance: 100,
+    maxPatternLength: 32,
+    minMatchCharLength: 1
+  };
+  var fuse = new Fuse(list, options); // "list" is the item array
```

Comment:
Instead of putting a comment about what list is, why don't we just give list a better name? I'm guessing that the contents of this is the file names, so I might call it `fileNames`.